packages/@aws-cdk/aws-rds/README.md

```diff
@@ -161,6 +161,12 @@ new rds.DatabaseCluster(stack, 'DatabaseCluster', {
 });
 ```
 
+## Copying Tags From the DB Instance to the DB Snapshot
```

Comment:
I don't think this property is worth an entire section in the ReadMe. Just add it with an inline comment to one of the code examples.